stat/bench_constants_test.go

```diff
@@ -0,0 +1,13 @@
+// Copyright ©2024 The Gonum Authors. All rights reserved.
```

Comment:
Why did this move?

Reply:
I wanted to reuse these constants in `BenchmarkROC` in `stat/roc_test.go`, but leaving them in `stat/moments_bench_test.go` felt inappropriate, so I decided to move them into a common place.

I'm not sure of this project's conventions w.r.t. this, though, so I'm happy to move them back and duplicate them if you'd like.

Reply:
Yeah, the move wasn't necessary, so let's revert that.

Reply:
Oh, sorry, I forgot to mention that I also introduced a new constant `empty`, which is only used in a different file, `stat/roc_test.go`. Any preferences on what I do with that constant? Leave it here, move it with the others to `stat/moments_bench_test.go`, or do something else entirely with it?

Reply:
Let's just add the empty to the source location where the original constants were and revert the file addition.

Reply:
@kortschak Okay, done!